internal/admin/album_coverage_test.go

```
@@ -0,0 +1,207 @@
+// against an eligible denominator. The CD album's optimize denominator
+// is zero — it is already at the CarPlay target — which is the case a
+// track-count denominator would have rendered as "0 of 1 missing".
+func TestAlbumGridCarriesCoveragePerAlbum(t *testing.T) {
+	srv, _, _ := newTestServer(t)
+	seedCoverageLibrary(t, srv.deps.Manifest)
+
+	_, albums := albumsPage(t, srv, "")
+	byTitle := map[string]albumCoverage{}
+	for _, a := range albums {
+		var cov albumCoverage
+		blob, _ := json.Marshal(a["variants"])
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)